catalog/pipeline/fbc/0.1/fbc.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: bundle-fbc-pipelinerun
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: fbc
+spec:
+  description: >-
+    Tekton pipeline to release HACBS Snapshot to Quay
+  params:
+    - name: snapshot
+      type: string
+      description: The Snapshot in JSON format
+    - name: enterpriseContractPolicy
+      type: string
+      description: JSON representation of the EnterpriseContractPolicy
+    - name: iibJsonBuildArgs
+      type: string
+      description: IIB JSON Arguments to add a new build
+    - name: iibUrl
```

Comment:
This could maybe use a default to prod. Probably the task too.

Reply:
I would add it just to one place - like here where you pointed. I don't like when you have the same default in several places then you never know the source of true.

Reply:
added in `fbc.yaml`.

Reply:
+1 to putting the default in just one spot

Reply:
Then it should probably go to the task that is the reusable piece.

Reply:
ok, moved to the task.